doc/usermanual/CMakeLists.txt

```diff
@@ -9,8 +9,10 @@ message(STATUS ${IMAGES_PATHS})
 
 if(BUILD_USERMANUAL)
 	add_custom_target(usermanual ALL)
+	add_custom_target(api ALL)
```

Comment:
Oh, and you only add the translated docs to the `api` target, I guess the regular ones should be built there, too?